robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowApplication.java

```diff
@@ -833,4 +836,21 @@ public void setSystemService(String key, Object service) {
     shadowContext.setSystemService(key, service);
   }
 
+  @Implementation
+  public void registerActivityLifecycleCallbacks(Application.ActivityLifecycleCallbacks callback) {
```

Comment:
I wonder if its better for testers to just test behavior rather than capture the listeners?